consensus/common_test.go

```diff
@@ -39,8 +39,8 @@ const (
 )
 
 // genesis, chain_id, priv_val
-var config *cfg.Config              // NOTE: must be reset for each _test.go file
-var ensureTimeout = time.Second * 1 // must be in seconds because CreateEmptyBlocksInterval is
+var config *cfg.Config                     // NOTE: must be reset for each _test.go file
+var ensureTimeout = time.Millisecond * 100 // must be in seconds because CreateEmptyBlocksInterval is
```

Comment:
I guess it was because of this: https://github.com/tendermint/tendermint/blob/0c9c3292c918617624f6f3fbcd95eceade18bcd5/consensus/mempool_test.go#L41. I will check this once more.